shell/packages/sandstorm-db/db.js

```diff
@@ -1010,20 +1010,22 @@ _.extend(SandstormDb.prototype, {
       return false;
     }
 
-    const googleDomain = this.getOrganizationGoogle();
-    const emailDomain = this.getOrganizationEmail();
+    const orgMembership = this.getOrganizationMembership();
+    const googleEnabled = orgMembership.google.enabled;
```

Comment:
Strictly speaking, the migration guarantees that `orgMembership` will exist and be in a well-defined structure.  But if you prefer to skip the migration if none of the org settings are set yet as your comment suggests below, then I suppose I'd need to guard here.

Another option would be to have `SandstormDb.prototype.getOrganizationMembership` guarantee that it returns a populated structure with the usual default values.